scripts/build.sh

```diff
@@ -22,7 +22,7 @@ fi
 if [[ "${TARGET_PLATFORM}" == win-* ]]; then
     conda install -y "nsis=3.01" -c conda-forge --override-channels
 fi
-pip install git+git://github.com/chrisburr/constructor@64ebd6d34f0f18684c76c0bebcfab41c38d55083#egg=constructor --force --no-deps
```

Comment:
This is probably why the last tag was failing.

Reply:
If we merge https://github.com/conda-forge/constructor-feedstock/pull/61 we don't need this pip install override, right?